### ISSUE TITLE
glucosedistribution.js seems to have an error in the code

### ISSUE THREAD

**Issue by zreptil** [nightscout/cgm-remote-monitor#4490]
**Describe the bug**
line 410 doesn't seem to make sense:
var GVIDelta = Math.floor(glucose_data[0].bgValue,glucose_data[glucose_data.length-1].bgValue);

Math.floor calculates the next lower integer number, but doesn't take multiple values. The result of this is the value of the first glucose value in the array. Is this the expected behavior?

This has influence at the value GVI on the page distributio in the reports.

**Comment by rob-andy** [comment]
This is a good pick up.  I agree 100% the correct code should be 
```javascript
var GVIDelta = Math.floor(glucose_data[0].bgValue,glucose_data[glucose_data.length-1].bgValue);
```

The delta should be the difference between the first and last reading

**Comment by jakobsandberg** [comment]
We can close this issue as it was resolved with #4502
